spec/asyncapi.md

```diff
@@ -1371,6 +1388,7 @@ Field Name | Type | Description
 
 This object MAY be extended with [Specification Extensions](#specificationExtensions).
 
+<!-- asyncapi-example-tester:{"name":"Message Trait Object","json_path":"$.components.messageTraits.commonHeaders"} -->
```

Comment:
This shouldn't be in this line but just before \```json``` tag

Reply:
Got it. Will check it out.

Reply:
resolved with ed55789

Reply:
fixed more via 10ad1ed